NuRadioReco/modules/trigger/radiant_aux_trigger.py

```diff
@@ -0,0 +1,173 @@
+import logging
+import time
+import numpy as np
+from NuRadioReco.framework.trigger import RadiantAUXTrigger
+from NuRadioReco.modules.base.module import register_run
+from NuRadioReco.framework.parameters import channelParameters as chp
+from NuRadioReco.modules.trigger.highLowThreshold import get_majority_logic
+from NuRadioReco.utilities import fft
+import matplotlib.pyplot as plt
+logger = logging.getLogger('radiant_aux_trigger')
+
+def tanh_func(x, b, c):
+    return 0.5*(np.tanh((x-b)/c) + 1)
+
+class triggerSimulator:
+    """
+    """
+
+    def __init__(self):
+        self.__t = 0
+        self.begin()
+
+    def begin(self):
+        return
+    
+    def get_filtered_trace_trigger(self, trace):
+        n_samples = len(trace)
+        spectrum = fft.time2freq(trace, self.sampling_rate)
+        freqs = np.fft.rfftfreq(n_samples, 1 / self.sampling_rate)
+        filtering_out = np.where( (freqs < 0.08) | (freqs > 0.2)) # or!
+        filtered_spectrum = spectrum.copy()
+        filtered_spectrum [filtering_out] *= 0.0
+        filtered_trace = fft.freq2time(filtered_spectrum, self.sampling_rate)
+
+        return filtered_trace
+
+    def get_threshold(self, trace, threshold_sigma, int_window):
+        noise_samples = int(130 * self.sampling_rate)
+        noise = self.get_power_int_trace(trace[10:noise_samples], int_window)
+        threshold = threshold_sigma * np.std(noise)
+        return threshold
+        
+    def get_power_int_trace(self, trace, int_window):
+        """
+        calculats a power integration over a given time window
+
+        Parameters
+        ----------
+        trace: array of floats
+            the signal trace
+        window: float
+            the integration window
+        Returns
+        -------
+        integrated power: array of floats
+
+        """
+        dt = 1. / self.sampling_rate
+        i_window = int(int_window / dt)
+        power = trace ** 2
+        int_power = np.convolve(power, np.ones(i_window, dtype=int), 'valid') * dt
+        return int_power
+
+    def get_trigger_response(self, trace, threshold_sigma, int_window):
+        """
+        check if the trace satisfies the trigger condition
+
+        Parameters
+        ----------
+        trace: array of floats
+            the signal trace
+
+        threshold: float
+            the threshold as value of sigma (a factor of snr)
+
+        window: float
```

Comment:
sorry, time :) I added it.

Reply:
Sorry you still didn't address the first fixing. The parameter is `int_winow` instead of `window`. I am looking now and this also applies to the `get_power_int_trace` function